src/components/dropdown/dropdown.jsx

```diff
@@ -29,11 +29,11 @@ class Dropdown extends PureComponent {
 
     return (
       <ul className="dropdown__actions" style={ style }>
-        {this.props.actions.map((action, index) => (
+        { this.props.actions.map((action, index) => (
           <li className="dropdown__action" key={ index } onClick={ action.action }>
```

Comment:
I'd suggest using `${index}_${action.label}` as `key`. Simply using `index` might not be unique and consistent enough.